### PR TITLE
Shorten liballoc vec resize intra-doc link

### PR DESCRIPTION
**Pull request by pickfire**
r? @jyn514

### CODE REVIEW

File: library/alloc/src/vec.rs [hunk 1]
```diff
@@ -1456,9 +1456,9 @@ impl<T> Vec<T> {
     /// If `new_len` is less than `len`, the `Vec` is simply truncated.
     ///
     /// This method uses a closure to create new values on every push. If
-    /// you'd rather [`Clone`] a given value, use [`resize`]. If you want
-    /// to use the [`Default`] trait to generate values, you can pass
-    /// [`Default::default()`] as the second argument.
+    /// you'd rather [`Clone`] a given value, use [`Vec::resize`]. If you
+    /// want to use the [`Default`] trait to generate values, you can
+    /// pass [`Default::default`] as the second argument.
```
Comment (jyn514) [inline library/alloc/src/vec.rs:1461]:
```suggestion
    /// pass [`Default::default()`] as the second argument.
```

**Review by jyn514 — COMMENTED** [review]
r=me with nit fixed

File: library/alloc/src/vec.rs (hunk 1 above)
Comment (pickfire) [reply library/alloc/src/vec.rs:1461]: I specifically removed that bracket because `Default::default` is passed instead of `Default::default` in the code.

Comment (jyn514) [reply library/alloc/src/vec.rs:1461]: Ah good point. This looks good then.

**Pull request merged**